src/pybind/ceph_argparse.py

```diff
@@ -1073,7 +1073,7 @@ def validate_command(sigdict, args, verbose=False):
                 break
 
         if not found:
-            print('no valid command found; 10 closest matches:', file=sys.stderr)
+            print('no valid command found; {0} closest matches:'.format(min(len(bestcmds), 10)), file=sys.stderr)
```

Comment:
to avoid repeating `10` here. we can also:
```py
bestcmds = bestcmds[:10]
print('no valid command found; {0} closest matches:'.format(len(bestcmds), file=sys.stderr)
for cmdsig in bestcmds:
  # ...
```

Reply:
Sounds good. Thanks for your review and advice @tchaikov